_conferences/droidcon-sf-2016.md

```diff
@@ -0,0 +1,12 @@
+---
+name: "Droidcon"
+website: http://sf.droidcon.com/2016
+location: San Francisco, USA
+
+date_start: 2016-03-00
+date_end:   2016-03-00
+
+cfp_start: 2015-00-00
```

Comment:
The CFP is technically still open, just running until ~6 weeks before the conference starts.